code/code/misc/combat.cc

```diff
@@ -3982,6 +3989,9 @@ int TBeing::oneHit(TBeing *vict, primaryTypeT isprimary, TThing *weapon, int mod
 	}
       }
 
+  // Add toughness 
+  doToughness(vict);
```

Comment:
indent

Reply:
Fixed it. The inconsistency of tab vs space in the code is annoying